### PR TITLE
Cesm xarray france

### PR DESCRIPTION
**Pull request by AlessandroLovo**
Better France shape for CESM:
for the before and after see issue #79

### CODE REVIEW

File: ERA/ERA_Fields_New.py [hunk 1]
```diff
@@ -1157,7 +1157,7 @@ def create_mask_xarray(model:str, area:str, lsm:xr.DataArray) -> xr.DataArray:
     if model in ['ERA5', 'CESM']:
         if area == 'France':
             mask = standardize_dim_names(lsm > 0.5) # convert to bool keeping only the land masses
-            mask *= (mask.lat < 52)*(mask.lat > 42)*(mask.lon > -5)*(mask.lon < 8.3) # identify the rough region
+            mask *= (mask.lat < 52)*(mask.lat > 42)*(mask.lon % 360 > -5 % 360)*(mask.lon % 360 < 8.3 % 360) # identify the rough region
```
Comment (georgemilosh) [inline ERA/ERA_Fields_New.py]: any reason why we need to check this criterion?

**Comment by georgemilosh** [comment]
I am a bit busy this month. How urgent is this pull request?

**Comment by AlessandroLovo** [comment]
For me there's no urgency to merge. Right now I am working on the branched repo.
When you have time just think if this will mess up your previous CESM work and decide whether to merge.
Maybe I could add the option to still use the old boxy France.

File: ERA/ERA_Fields_New.py (hunk 1 above)
Comment (AlessandroLovo) [reply ERA/ERA_Fields_New.py]: This way of doing it didn't work. But the point is that when you say `lon > -5` you are assuming that lon is in [-180,180], and if it is not (e.g. in [0,360]) lon > -5 will be the same as `lon > 0`. Not good.

Anyways, this is properly fixed in the new version

**Pull request merged**